src/scripts/allocation_receipts.py

```diff
@@ -0,0 +1,199 @@
+               f"Community Details\n  {poap_details}"
+
+
+def load_all_data_from_out(allocation_files: AllocationFiles) -> AllocationDetails:
+    # Load Holder Info
+    holder_details = HolderDetails(
+        data=index_by_account(
+            CombinedGnoHolder.load_from(
+                load_from=allocation_files.holder_data.combined
+            )
+        ),
+        allocations=IndexedAllocations.load_from_file(
+            file=allocation_files.holder_allocation
+        )
+    )
+
+    # Load Trader Info
+    raw_trader_data = CowSwapTrader.load_and_merge_network_trader_data(
+        mainnet_file=allocation_files.trader_data.traders.filename('mainnet'),
+        gchain_file=allocation_files.trader_data.traders.filename('gchain'),
+    )
+    primary_trader_details = TraderDetails(
+        data=CowSwapTrader.load_from(
+            load_file=allocation_files.trader_data.primary_trader
+        ),
+        allocations=IndexedAllocations.load_from_file(
+            file=allocation_files.trader_data.primary_allocation
+        ),
+        allocation_type=AllocationType.PRIMARY
+    )
+    consolation_details = TraderDetails(
+        data=CowSwapTrader.load_from(
+            load_file=allocation_files.trader_data.consolation_trader
+        ),
+        allocations=IndexedAllocations.load_from_file(
+            file=allocation_files.trader_data.consolation_allocation
+        ),
+        allocation_type=AllocationType.CONSOLATION
+    )
+
+    # Load POAP Info
+    poap_details = PoapDetails(
+        # TODO - load gchain alphas
+        mainnet=Account.load_from(
+            load_file=allocation_files.alpha_traders.filename('mainnet')
+        ),
+        gchain=Account.load_from(
+            load_file=allocation_files.alpha_traders.filename('gchain')
+        ),
+        allocations=IndexedAllocations.load_from_file(
+            file=allocation_files.poap_allocations
+        )
+    )
+
+    return AllocationDetails(
+        holder=holder_details,
+        primary=primary_trader_details,
+        consolation=consolation_details,
+        poap=poap_details,
+        raw_trader_data=raw_trader_data
+    )
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description="Fetch Allocation Receipts for a list of accounts"
+    )
+    parser.add_argument(
+        "--account-file",
+        type=str,
+        help="file containing ethereum addresses",
+    )
+    args = parser.parse_args()
+    # Load all Data related to allocations and where they were derived from
+
+    allocation_details = load_all_data_from_out(AllocationFiles())
+
+    accounts = sorted(
+        Account.load_from(File(name=args.account_file, path='./out/')),
+        key=lambda t: t.account
+    )
+
+    receipts = [allocation_details.account_detail_string(a.account) for a in accounts]
+    with open(File('allocation-receipts.csv').filename(), 'w', encoding='utf-8') as out:
+        out.write("\n".join(receipts))
```

Comment:
Have you considered if printing is maybe more useful for the user of the script (it's trivial for a user to pipe the output into a file)?

Reply:
I was printing the results in a commit just before the most recent commit, but piping the results to file is a great suggestion. I think I may do that.